src/tests/fixtures/users.fixture.js

```diff
@@ -15,9 +15,14 @@ export const signupFixtures = {
 
 export const signedUpUserToken = JwtService.generateToken({ email: signupFixtures.email });
 export const unregisteredEmail = {
-  email: 'higustave123@gmail.com'
+  email: 'higustave123346432@gmail.com'
```

Comment:
Always use randomly generated value. I've said this a couple of times.

Reply:
Sorry for unimplementation. Will not happen again.